nearcore/src/config.rs

```diff
@@ -458,6 +462,8 @@ pub struct Config {
     pub db_migration_snapshot_path: Option<PathBuf>,
     /// Different parameters to configure/optimize underlying storage.
     pub store: near_store::StoreConfig,
+    #[serde(default = "default_log_config")]
+    pub log_config: Option<PathBuf>,
```

Comment:
I’d personally just hard-code the path and not add this option.

But if this is added it should be called `log_config_file` or something.

Also, perhaps introduce `struct LogConfig` which would contain this and `log_summary_style` fields? I wouldn’t even worry about flattening it.

Reply:
Agree with hard-coding.

Don't understand how or why to move `log_summary_style` to a different structure. `log_summary_style` is already a field in `Config`, and moving it will be complicated.